app/scripts/actions/links.js

```diff
@@ -43,6 +44,8 @@ export function updateURL() {
       });
     }
 
+    if (locationFilter) query += `&coreDatasetsLocation=${locationFilter}`;
```

Comment:
I think that we should manage the URL with a library (That's a thing that you are always telling me)... https://www.npmjs.com/package/query-string